### PR TITLE
Feature/works about us section

### PR DESCRIPTION
**Pull request by ishimwepati**
This is Work Section and About Section, it include 👍 

- Use of Grid and Flex
- nav 
- css Colors
- Buttons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
...
+            <nav class="grid-item last">
+                    <button class="button1" type="button">See project</button>
+            </nav>
+        </nav>
+        
+    </section>
+    <section class="about">
+            <nav class="grid-container-about">
+                <nav class="First-image">
+                    <img src="icon/illustration 1- about me-mobile.svg" class="upper_img">
```
Comment (Tufahel) [inline index.html]: - [ ] This image is a background image. Best practice to add a background image is to use `background-image` property of CSS, not the HTML `img` tag. Kindly fix this issue.

File: index.html [hunk 2]
```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
...
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-aboutdb"><h4>Database Managment</h4> </nav>
+                    <nav class="flex-about flex-aboutvs"><h4>Visual Control</h4></nav>
+                </nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-aboutcli"><h4>CLI</h4> </nav>
+                    <nav class="flex-about flex-aboutweb"><h4>Webdevelopment</h4></nav>
+                    <nav class="flex-about flex-aboutapi"><h4>API Design</h4></nav>
+                </nav>
+            </nav>
```
Comment (Tufahel) [inline index.html]: - I can see you are using `nav` tag everywhere. It is a serious semantic error. `nav` tag is specifically for `navigation menu` of an webpage.
- There are other dedicated tags for specific sections such as `div`, `ul`, `li`.
- [x] Kindly use dedicated `tags` for each section.

***[Hints]*** I have mentioned some corrections for `ul` and `li` tags. Other than `ul` and `li` tags replace all `nav` tags with `div` tag.

File: index.html [hunk 3]
```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
...
+            <nav class="grid-item main">
+                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
+            </nav>
+            
+            <nav class="grid-item flex-container">
+                <nav class="flex-item flex-item1"><h4>Ruby on Rails</h4> </nav>
+                <nav class="flex-item flex-item2"><h4>CSS</h4></nav>
+                <nav class="flex-item flex-item3"><h4>JavaScrit </nav>
+                <nav class="flex-item flex-item4"><h4>HMTL</h4></nav>
+            </nav>
```
Comment (Tufahel) [inline index.html]: - [x] This is an unordered list section. For list items, it is best to use `ul` and `li` tags. `nav` tag is specific for `navigation menu`.
- As Example:
`<ul>`
`  <li></li>`
`  <li></li>`
`</ul>`

File: index.html [hunk 4]
```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
...
```
Comment (Tufahel) [inline index.html]: - [ ] This is an unordered list section. For list items, it is best to use `ul` and `li` tags. `nav` tag is specific for `navigation menu`.
- As Example:
`<ul>`
`  <li></li>`
`  <li></li>`
`</ul>`

File: index.html [hunk 5]
```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
...
```
Comment (Tufahel) [inline index.html]: - [ ] This is an unordered list section. For list items, it is best to use `ul` and `li` tags. `nav` tag is specific for `navigation menu`.
- As Example:
`<ul>`
`  <li></li>`
`  <li></li>`
`</ul>`

File: index.html [hunk 6]
```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
...
+                    <img src="icon/icon-languages.svg">
+                    
+                </nav>
+                <nav class="title"><h3>Languages</h3></nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-about1"><h4>JavaScript</h4> </nav>
+                    <nav class="flex-about flex-about2"><h4>Ruby</h4></nav>
+                    <nav class="flex-about flex-about3"><h4>HTML</h4></nav>
+                    <nav class="flex-about flex-about4"><h4>CSS</h4></nav>
+                </nav>
```
Comment (Tufahel) [inline index.html]: - [x] This is an unordered list section. For list items, it is best to use `ul` and `li` tags. `nav` tag is specific for `navigation menu`.
- As Example:
`<ul>`
`  <li></li>`
`  <li></li>`
`</ul>`

File: index.html [hunk 7]
```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
...
+                <nav class="grid-language">
+                    <img src="icon/icon-frameworks.svg">
+                    
+                </nav>
+                <nav class="title"><h3>Frameworks</h3></nav>
+                <nav class="grid-about flex-container-about">
+                    <nav class="flex-about flex-aboutjs"><h4>React.js</h4> </nav>
+                    <nav class="flex-about flex-about2"><h4>Ruby on Rails</h4></nav>
+                    <nav class="flex-about flex-about3"><h4>RSpec</nav>
+                </nav>
```
Comment (Tufahel) [inline index.html]: - [ ] This is an unordered list section. For list items, it is best to use `ul` and `li` tags. `nav` tag is specific for `navigation menu`.
- As Example:
`<ul>`
`  <li></li>`
`  <li></li>`
`</ul>`

File: style.css [hunk 8]
```diff
@@ -74,3 +74,371 @@ li {
...
+
+.button1 {
+  color: black;
+  border: 2px solid #4caf50;
+  background-color: #36b37f;
+  border-radius: 2px;
+  width: 122px;
+  height: 48px;
+  font-size: 17px;
+}
```
Comment (Tufahel) [inline style.css]: - [ ] Kindly implement the button correctly to match the given design. The `background color` of the button and `text color` is not matching with the given design.
- [ ] Also implement the button interactions such as (`hover`, `click`, `disable`)
- Here is the button interactions instruction:
- image

- ***Your Design***
- image
- ***Given Design***
- image

File: style.css [hunk 9]
```diff
@@ -74,3 +74,371 @@ li {
...
+.button-resume {
+  color: black;
+  border: 2px solid #4caf50;
+  background-color: #36b37f;
+  border-radius: 2px;
+  width: 150px;
+  height: 48px;
+  font-size: 17px;
+  margin-top: 7%;
+}
```
Comment (Tufahel) [inline style.css]: - [ ] Kindly implement the button correctly to match the given design. The `background color` of the button and `text color` is not matching with the given design.
- [ ] Also implement the button interactions such as (`hover`, `click`, `disable`)
- Here is the button interactions instruction:
- image

- ***Your Design***
- image
- ***Given Design***
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @ishimwepati :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Linter checks are passing.
✔️ Follow the correct GitHub flow.
✔️ Commit messages are meaningful.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ishimwepati) [reply index.html]: Thanks a lot, I did it!

File: index.html (hunk 2 above)
Comment (ishimwepati) [reply index.html]: Great

Comment (ishimwepati) [reply index.html]: Thanks a lot

File: index.html (hunk 3 above)
Comment (ishimwepati) [reply index.html]: Done

Comment (ishimwepati) [reply index.html]: Great

File: index.html (hunk 4 above)
Comment (ishimwepati) [reply index.html]: Good

File: index.html (hunk 5 above)
Comment (ishimwepati) [reply index.html]: Good

File: index.html (hunk 6 above)
Comment (ishimwepati) [reply index.html]: Good

**Review by Bantealem — APPROVED** [review]
# Status: Approved
Hi @ishimwepati  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] Correct GitHub flow.
- [x] No Linters error
- [x] Professional README
- [x] Descriptive PR
## Optional suggestions
- [ ] Please make sure to implement the `hover`, `click` and `passed` properties of each button for your portfolio. It will show whether you are touching the button or not.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**